apps/etterlatte-saksbehandling-ui/client/src/components/tilbakekreving/vurdering/TilbakekrevingVurderingSkjema.tsx

```diff
@@ -26,6 +26,7 @@ import { JaNei, JaNeiRec } from '~shared/types/ISvar'
 import { useForm } from 'react-hook-form'
 import { ControlledDatoVelger } from '~shared/components/datoVelger/ControlledDatoVelger'
 import { ControlledRadioGruppe } from '~shared/components/radioGruppe/ControlledRadioGruppe'
+import { throttle } from 'lodash'
 
 const initialVurdering = {
```

Comment:
Dette funker ikke.
TS7053: Element implicitly has an any type because expression of type string can't be used to index type TilbakekrevingVurdering
No index signature with a parameter of type string was found on type TilbakekrevingVurdering